sqlalchemy_continuum/manager.py

```diff
@@ -431,6 +431,21 @@ def track_cloned_connections(self, c, opt):
                 if not connection.closed and connection.connection is c.connection:  # ConnectionFairy is the same - this is a clone
                     self.units_of_work[c] = uow
 
+    def get_python_processed_parameter(self, parameter, column, dialect):
+        param_transformer = column.type.dialect_impl(dialect)
+        process = param_transformer.result_processor(dialect, column.type)
+        if process is not None:
+            # https://github.com/sqlalchemy/sqlalchemy/blob/231262d4782fc4d1a5d5fdbf985a06e04270e878/lib/sqlalchemy/sql/type_api.py#L452
+            return process(parameter)
+        return parameter
+
+    def process_parameters(self, table_name, parameters, dialect):
```

Comment:
(Opinionated) We don't really need 2 functions here - we can do it in a single function with a for loop ?

Reply:
yeah sure we can do that, kept them separate as method `get_python_processed_parameter` uses dialect to process parameters so if any dialect specific condn (process being None, not aware of any other cond's as of now) is identified the handling can be dealt within that method itself